apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -356,14 +361,14 @@ export class CaseNotificationService extends BaseNotificationService {
         formatDefenderRoute(this.config.clientUrl, theCase.type, theCase.id),
     )
 
-    return this.sendEmail(
+    return this.sendEmail({
       subject,
-      body,
-      theCase.defenderName,
-      theCase.defenderEmail,
-      undefined,
-      !theCase.defenderNationalId,
-    )
+      html: body,
+      recipientName: theCase.defenderName,
+      recipientEmail: theCase.defenderEmail,
+      attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.